### DISCUSSION TITLE
How to add Qt and osg libraries to include path

### DISCUSSION THREAD

**Discussion by rileymjohnson** [omnetpp/omnetpp#995] [Q&A] [answered]
I am looking to write a custom figure render that relies on some Qt stuff. However, this is not possible to do within my project as I cannot include the necessary Qt headers. How should I go about adding to the include path? Also, it's important that eclipse recognizes the updated include path as well. I am on windows using mingw/msys2. I found the folder that contains the necessary headers (`tools/win32.x86_64/opt/mingrw64/include`), but I'm not sure what to do with it. Also, the version that I'm using is: `omnetpp-7.0.0pre2`.

**Comment by rhornig** [comment, marked as answer]
`QT_CFLAGS` and `QT_LIBS` variables are available in `Makefile.inc` so you can use those variables in a `makefrag` file to add them to  `CFLAG` and `LDFLAGS`
